pkg/controller/gather_job.go

```diff
@@ -79,6 +78,7 @@ func (d *GatherJob) Gather(ctx context.Context, kubeConfig *rest.Config, protoKu
 		return err
 	}
 
-	klog.Warning("stopped")
```

Comment:
OT - I am still wondering what's the purpose of this in our normal execution..... :)

Reply:
No idea either, its nice to know that the `operator.go` finished ie. started every necessary goroutine for IO to work, but why is it this simple warning is a bit weird.

Maybe in a bygone age, the `operator.go` shouldn't have stopped :)